lib/users/data.js

```diff
@@ -31,4 +32,27 @@ const modelMerged = (userModel) => {
   return contentTypes([userConfig]);
 };
 
+/*
+ * Cleaner user data post form submission
+ *
+ * @param {object} body req.body post-form submission
+ *
+ * @returns {object} cleaned version of form data
+ */
+const cleanFormData = (body) => {
```

Comment:
Why does this need to exist? Why can't you just use the form field names? And why is the bcrypt hash part of cleaning up the form data?